src/servers/http_server_v2.cc

```diff
@@ -868,9 +868,16 @@ WriteDataToJson(
 void
 EVBufferAddErrorJson(evbuffer* buffer, TRITONSERVER_Error* err)
 {
+  rapidjson::Document response;
+  response.SetObject();
   std::string message = std::string(TRITONSERVER_ErrorMessage(err));
-  std::string message_json = "{ \"error\" : \"" + message + "\" }";
-  evbuffer_add(buffer, message_json.c_str(), message_json.size());
+  rapidjson::Value message_json(
+      rapidjson::StringRef(message.c_str(), message.size()),
+      response.GetAllocator());
```

Comment:
You don't have to explicitly put message_json in the 'response'? Just using the 'response' allocator causes it to be added?

Reply:
Why did you remove "error" field from the json response? Doesn't the client expect
`{error: <error_message>}` and not `<error_message>`
Also to answer your question David, when an allocator is passed to rapidjson:Value() it creates a copy of the string. When the allocator is not passed it uses the address of the string itself and does not create a copy of it.

Reply:
Protocol requires a json response that is { "error" : <msg> }, so that part is correct. I just don't see where message_json gets added to 'response'. Perhaps I'm just missing it.

Reply:
Tanmay had missed a line that does exactly that -> `response.AddMember("error", message_json, response.GetAllocator());`
I fixed it

Reply:
My bad. The line got accidentally dropped while committing. Thanks @CoderHam for fixing it.